packages/filebrowser-extension/schema/browser.json

```diff
@@ -131,11 +131,6 @@
   "title": "File Browser",
   "description": "File Browser settings.",
   "jupyter.lab.shortcuts": [
-    {
```

Comment:
Moved backspace handling into the DirListing class. You can reason from the way it's configured here that it doesn't work if the directory is empty, because if it's empty there is no `.jp-DirListing-itemText` node.